console/src/assets/i18n/de.json

```diff
@@ -52,7 +52,7 @@
             "DETAIL": "Detail",
             "CREATE": "Erstellen",
             "MY": "Meine Informationen",
-            "LOGINNAMES":"Loginnamen",
+            "LOGINNAMES":"Zugangsmethoden",
```

Comment:
```suggestion
            "LOGINNAMES":"Benutzernamen",
```

@fgerschwiler do you have a better idea?

Reply:
or is it `Loginname`?

Reply:
@fgerschwiler would name it 'Loginnamen'